open-company/open-company-web
# Design feedback and section sort

cards:
- https://trello.com/c/NlXTc1vJ
- https://trello.com/c/LaMcOYie

Test:
- check the punch list of the first card
- update a section and check that it's not moved at the top
- refresh the page and check that the previously updated section is still in the same position
